doc/whats_new/v0.20.rst

```diff
@@ -274,6 +274,10 @@ Preprocessing
 - :class:`preprocessing.PowerTransformer` and
   :func:`preprocessing.power_transform` ignore and pass-through NaN values.
   :issue:`11306` by :user:`Guillaume Lemaitre <glemaitre>`.
+- :class:`preprocessing.RobustScaler` and :func:`preprocessing.robust_scale`
+  ignore and pass-through NaN values. In addition, the scaler can now be fitted
```

Comment:
I think the second comment is a separate enhancement and should be reported separately.

I would like to see the "ignore and pass through NaN values" what's news merged into one (not necessarily in this PR).